doc/classes/ClassDB.xml

```diff
@@ -65,6 +65,15 @@
 				Returns an array with the names all the integer constants of [param class] or its ancestry.
 			</description>
 		</method>
+		<method name="class_get_method_argument_count" qualifiers="const">
+			<return type="int" />
+			<param index="0" name="class" type="StringName" />
+			<param index="1" name="method" type="StringName" />
+			<param index="2" name="no_inheritance" type="bool" default="false" />
+			<description>
+				Returns the number of arguments of the method [param method] of [param class] or its ancestry if [param no_inheritance] is [code]false[/code].
```

Comment:
So many "of"s here. Perhaps replacing one would still make sense?
```suggestion
				Returns the number of arguments for the method [param method] of [param class] or its ancestry if [param no_inheritance] is [code]false[/code].
```

Reply:
I'm not so sure `for` is grammatical here, it is the arguments *of* the method, but will see what I can do with the wording